csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -1516,6 +1728,22 @@ Valid values are:
     not_contacted
     open
 
+Each status is mutually exclusive - only one status is valid for a particular vulnerability at a particular time. As the vulnerability ages, a party’s involvement could move from state to state. However, in many cases, a document producer may choose not to issue CSAF documents at each state, or simply omit this element altogether. It is recommended, however, that vendors that issue CSAF documents indicating an open or in-progress Involvement should eventually expect to issue a document as `disputed` or `completed`.
+
+> The two vulnerability involvement status states, `contact_accepted` and `not_contacted` are intended for use by document producers other than vendors (such as research or coordinating entities).
+
+The value `completed` indicates that the vendor asserts that investigation of the vulnerability is complete. No additional information, fixes, or documentation from the vendor about the vulnerability should be expected to be released.
+
+The value `contact_accepted` indicates that the document producer attempted to contact the affected vendor.
```

Comment:
Should this value be `contact_attempted`? I also changed the description so that it is more neutral. Please also change line 1725 and the schema.
```suggestion
The value `contact_attempted` indicates that the document producer attempted to contact the party, but did not get a response yet.
```

Reply:
Fixed by edits on the branch